assignments/prototypes.js

```diff
@@ -20,10 +19,13 @@ function GameObject(input) {
   this.createdAt = input.createdAt;
   this.dimensions = input.dimensions;
 }
-
 Humanoid.prototype.destroy = function() {
```

Comment:
your destroy function isn't working here because you are calling the object Humanoid instead of GameObject

Reply:
Otherwise, this is the correct syntax